2016/linear-transformations/spanish/sentence_translations.json

```diff
@@ -576,7 +576,7 @@
   ]
  },
  {
-  "translatedText": "Y a riesgo de ser redundante aquí, descubrir cómo una cizalla transforma un vector dado se reduce a multiplicar esta matriz por ese vector.",
+  "translatedText": "Y con riesgo de ser redundante aquí, descubrir cómo una cizalla transforma un vector dado se reduce a multiplicar esta matriz por ese vector.",
```

Comment:
That's an interesting one, which might have something to do with how we speak Spanish in different parts of the world. As far as I know, in Spain we tend to say `A riesgo de` rather than `Con riesgo de`, is there a difference in other regions? I'd be curious to find out :)

Reply:
Sure! In latam, `Con riesgo de` is usually used as a more natural way to say `A riesgo de.` However, `A riesgo de` is also valid, so at least in latam, both would be correct. However, if you think it would be easier or better to use `a riesgo de` then I see no problem in changing it.

Reply:
Ahh, ok, `Con` is perfect for me, don't worry :) If I ever correct something and it is something common in latam don't hesitate to tell me so, I rather have a latam version than an spaniard one, since I believe the number of spectators is way bigger on the other side of the ocean. And if I do a big `españolada` tell me :)